nemo/collections/nlp/modules/common/text_generation_strategy.py

```diff
@@ -486,6 +487,28 @@ def neva_process_prompts(prompt, tokenizer, multimodal_cfg, num_media_latents, c
             copy.deepcopy(list_data_dict), multimodal_cfg, num_media_latents
         )  # HARDCODED FOR NOW
         data_dict = preprocess_llama_2(sources, tokenizer, multimodal_cfg)
+    elif multimodal_cfg["conv_template"] == "yi_34b":
```

Comment:
why not just yi?

Reply:
there are versions of yi that use slightly different template